homeassistant/components/synology_dsm/__init__.py

```diff
@@ -479,8 +548,6 @@ def __init__(
         entity_info: Dict[str, str],
     ):
         """Initialize the Synology DSM entity."""
-        super().__init__()
```

Comment:
This should forward any remaining parameters to the parent class, eg the coordinator entity.